### PR TITLE
Documentation -- Function Headers -- sys_signal

### PR DESCRIPTION
**Pull request by Soldia1138**
sys_signal of #107

### CODE REVIEW

**Review by jonpas — APPROVED** [review]

File: addons/sys_signal/fnc_handleSignalReturn.sqf [hunk 1]
```diff
@@ -1,16 +1,16 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Return function for signal calculation. Will be called after signal has been calculated in extension
```
Comment (Sniperhid) [inline addons/sys_signal/fnc_handleSignalReturn.sqf]: in *the* extension

**Review by Sniperhid — CHANGES_REQUESTED** [review]

File: addons/sys_signal/fnc_getSignal.sqf [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+ * 2: Receiving Radio ID <STRING>
+ * 3: Transmitting Radio ID <STRING>
  *
  * Return Value:
- * RETURN VALUE <TYPE>
+ * Tuple of power and maximum signal strength <ARRAY>
  *
  * Example:
- * [ARGUMENTS] call acre_COMPONENT_fnc_FUNCTIONNAME
+ * [FREQUENCY, POWER, "_radioID1", "_radioID2"] call acre_sys_signal_fnc_getSignal
```
[the remark this answers is not in the document]
Comment (jonpas) [reply addons/sys_signal/fnc_getSignal.sqf]: Preferably those examples should have an example of direct data input.

**Comment by Soldia1138** [comment]
Deleted the example and corrected the wording on return-function

**Review by Sniperhid — APPROVED** [review]
LGTM

File: addons/sys_signal/fnc_handleSignalReturn.sqf [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
  *
  * Arguments:
- * 0: ARGUMENT ONE <TYPE>
- * 1: ARGUMENT TWO <TYPE>
+ * 0: Radio IDs (TX and RX) <ARRAY>
+ * 1: Antenna and Signal Strength <ARRAY>
  *
  * Return Value:
- * RETURN VALUE <TYPE>
+ * -
```
Comment (jonpas) [inline addons/sys_signal/fnc_handleSignalReturn.sqf]: `None`

File: addons/sys_signal/fnc_handleSignalReturn.sqf [hunk 4]
```diff
@@ -1,16 +1,14 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Return function for signal calculation. Will be called after signal has been calculated in the extension
```
Comment (jonpas) [inline addons/sys_signal/fnc_handleSignalReturn.sqf]: Should be dot at the end, so it'll look nice when we generate the functions list (this one won't be there, but still, consistency).

File: addons/sys_signal/fnc_getSignal.sqf [hunk 5]
```diff
@@ -1,22 +1,24 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Wrapper function for calling the signal calculation (extension)
```
Comment (jonpas) [inline addons/sys_signal/fnc_getSignal.sqf]: Final dot too.

**Review by jonpas — CHANGES_REQUESTED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**